BlueprintUICommonControls/Sources/Box.swift

```diff
@@ -47,7 +47,7 @@ public struct Box: Element {
                 if self.backgroundColor != view.backgroundColor {
```

Comment:
Looks like there was a merge race with us landing the formatting on the changes in this file vs the corner rounding changes – so ran the formatter; otherwise CI fails.